src/Components/Authentication/Desktop/ModalManager.tsx

```diff
@@ -33,12 +37,14 @@ export class ModalManager extends Component<
   state = {
     currentType: null,
     copy: null,
+    redirectTo: "/",
   }
 
   openModal = (options: ModalOptions) => {
     this.setState({
       currentType: options.mode,
       copy: options.copy,
+      redirectTo: options.redirectUrl || "/",
```

Comment:
I think the default we want here is `/personalize`?

Reply:
Got it, ok sounds good

Reply:
Actually I noticed this over in Force, but I guess the default we want here is `/` for login modes and `/personalize` for signups so more of a conditional?